src/index.html

```diff
@@ -16,16 +17,19 @@
       href="style.css"
     />
   </head>
+
   <body>
     <input
+      class="search-bars search-bars-big"
       type="text"
-      data-qa="keypress"
+      data-qa="big"
       placeholder="Try “Los Angeles“"
     />
 
     <input
+      class="search-bars search-bars-small"
       type="text"
-      data-qa="hover"
+      data-qa="small"
       placeholder="Try “Los Angeles“"
     />
```

Comment:
The input fields are missing the 'id' attribute. The 'id' attribute is used to associate the input field with a label for accessibility purposes. Please, add the 'id' attribute to your input fields.